### ISSUE TITLE
Proposal ref(controller): disable swap usage if there is a memory limit

### ISSUE THREAD

**Issue by aledbf** [deis/deis#3205]
If the app uses `deis limits:set -m type=XXX -a XXX` add the flag `--memory-swap=-1`
```
docker help run
....
  --memory-swap=""           Total memory usage (memory + swap), set '-1' to disable swap (format: <number><optional unit>, where unit = b, k, m or g)
```

without that if the node uses swap it will not use the specified limit

**Comment by carmstrong** [comment]
I think disabling swap is reasonable. Is this in Docker 1.4.1 which we're on now?

**Comment by aledbf** [comment]
@carmstrong no, it's available since 1.5.0

**Comment by carmstrong** [comment]
> no, it's available since 1.5.0

Let's revisit this once we get that in stable (should be soon).

**Comment by azurewraith** [comment]
I think I ran into the side effects of this yesterday...
